src/pages/sung/LoginSung/LoginSung.js

```diff
@@ -1,8 +1,65 @@
 import React from 'react';
 import './LoginSung.scss';
+import { useState } from 'react';
+import InputLoginSung from './InputLoginSung';
+import { useNavigate } from 'react-router-dom';
 
 function LoginSung() {
-  return;
+  const [idValue, setIdValue] = useState('');
+  const [pwValue, setPwValue] = useState('');
+
+  const navigate = useNavigate();
+
+  const handleIdInput = e => {
+    setIdValue(e.target.value);
+  };
+  const handlePwInput = e => {
+    setPwValue(e.target.value);
+  };
+
+  const condition = idValue.includes('@') && pwValue.length > 5;
+
+  const goToMain = () => {
+    fetch('http://10.58.5.208:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        first_name: 'Sung',
+        last_name: 'Kang',
+        email: idValue,
+        password: pwValue,
+        phone_number: '0102145325325',
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        console.log('결과: ', result);
+        if (result.message === 'SUCCESS') {
+          navigate('/main-sung');
+        } else if (result.message === 'KEY_ERROR') {
+          alert('Key Error!');
+        } else if (result.message === 'INVALID_USER') {
+          alert('Invalid user!');
+        }
```

Comment:
에러 메시지들을 객체로 만들고 alert로 보여줄 내용을 mapping하면 좀 더 깔끔하게 관리할 수 있음!
```js
const ERROR_MESSAGE = {
  KEY_ERROR : "Key Error!",
  INVALID_USER : "Invalid user!",
}
```
```suggestion
        if(!result.token) {
          alert(ERROR_MESSAGE[result.message]);
          return;
        }
        
        navigate('/main-sung');
```